airbyte-workers/src/test/java/io/airbyte/workers/temporal/sync/SyncWorkflowTest.java

```diff
@@ -423,4 +450,13 @@ private static void verifyDbtTransform(final DbtTransformationActivity dbtTransf
         operatorDbtInput);
   }
 
+  private static void verifyShouldRefreshSchema(final RefreshSchemaActivity refreshSchemaActivity) throws IOException {
```

Comment:
I don't think that having method to wrap one line is helping. It is only call once. We should add a level of indirection if it is not needed.

Reply:
+1

Reply:
I updated this to be called multiple times in other tests

Reply:
For a one line, I still don't think that having on indirection is needed. It doesn't really with code readability and just makes you wondering what is in the implementation.

Reply:
The reason I did it was this was the pattern we seem to be using for syncWorkflowTests - all the activities are in separate methods just testing one method call

Reply:
Some other methods factorize the call of some static variables. Which is not the case here. Also what was done in the past might not have been the best. I would have the same comment for verifyPersistState function but it's already committed. I don't want to block the PR for that but I think that we should stick to the existing pattern if we think it is not the best. @jdpgrailsdev @colesnodgrass any though on that?